src/views/queries.tsx

```diff
@@ -0,0 +1,52 @@
+import Html from '@kitajs/html'
+import { singleton } from 'tsyringe'
+import { Page } from './common.js'
+
+@singleton()
+export default class QueriesTemplates {
+  constructor() {}
+  public chooseQueryPage = () => {
+    return (
+      <Page
+        title="Veritable - Queries"
+        heading="Queries"
+        headerLinks={[{ name: 'Queries', url: '/queries' }]}
+        stylesheets={['new-invite.css']}
+      >
+        <div class="main connections" hx-get="/queries"></div>
+        <div>
+          <h1 class="query-page-header">Queries</h1>
+          <hr class="divider"></hr>
+        </div>
+        <div class="query-container">
+          <div class="query-item" onclick="location.href='#';">
```

Comment:
These really should be anchor elements `<a>` not divs. The reaosn is that an `<a>` represents the semantics of what this element does in a way that a generic container element `<div>` does not.

I may have missed this in the homepage review btw so sorry.